client/packages/invoices/src/OutboundShipment/DetailView/modals/TaxEdit.tsx

```diff
@@ -28,12 +29,20 @@ export const TaxEdit = ({ disabled = false, tax, update }: TaxEditProps) => {
       />
       {/* Unmount when closing to reset state */}
       {modalController.isOn && (
-        <NonNegativeNumberInputModal
-          max={100}
+        <InputModal
           isOpen={modalController.isOn}
           onClose={modalController.toggleOff}
-          onChange={value => update(NumUtils.round(value, 2))}
-          initialValue={NumUtils.round(tax, 2)}
+          Input={
+            <NumericTextInput
```

Comment:
this tax edit modal behaves differently - it allows entering this sort of thing

<img width="550" alt="Screenshot 2024-01-24 at 4 07 20 PM" src="https://github.com/msupply-foundation/open-msupply/assets/9192912/15d2590a-6fcc-4dd6-90e5-4076a2c023f2">

which then results in this when you save and re-open

<img width="566" alt="Screenshot 2024-01-24 at 4 04 40 PM" src="https://github.com/msupply-foundation/open-msupply/assets/9192912/800ee416-4ad5-42ad-a611-5890141d8a7b">

the current numeric input does not allow the invalid numbers